pkg/apis/extensions/types.go

```diff
@@ -482,6 +471,23 @@ type DaemonSetStatus struct {
 	// ObservedGeneration is the most recent generation observed by the daemon set controller.
 	// +optional
 	ObservedGeneration int64
+
+	// UpdatedNumberScheduled is the total number of nodes that are running updated
+	// daemon pod
+	// +optional
+	UpdatedNumberScheduled int32
+
+	// NumberAvailable is the number of nodes that should be running the
+	// daemon pod and have one or more of the daemon pod running and
```

Comment:
Maybe remove "running" since a ready/available pod must be running? Just trying to make it more understandable:

```go
// NumberAvailable is the number of nodes that should be running the
// daemon pod and have at least one available daemon pod (ready for
// at least minReadySeconds)
```